index.html

```diff
@@ -5045,9 +5045,14 @@ <h4>Plain HTML or Polyfill DOM Result of the MathML Quadratic Formula</h4>
 			<rdef>meter</rdef>
 			<div class="role-description">
 				<p>An <a>element</a> that represents a scalar measurement within a known range, or a fractional value. See related <rref>progressbar</rref>.</p>
-				<p>Authors MUST NOT use the <code>meter</code> role to represent a value whose minimum and maximum values are unknown. In addition, authors MUST use <pref>aria-valuemin</pref> and <pref>aria-valuemax</pref> to make the minimum and maximum values available to assistive technologies.</p>
+				<p>Authors MAY set <pref>aria-valuemin</pref> and <pref>aria-valuemax</pref> to make the minimum and maximum values available to assistive technologies. Otherwise, their implicit values follow the same rules as <code>&lt;input[type="range"]&gt;</code> in [[HTML]]:</p>
+				<ul>
+					<li>If <code>aria-valuemin</code> is missing or not a <a href="#valuetype_number">number</a>, it defaults to 0 (zero).</li>
+					<li>If <code>aria-valuemax</code> is missing or not a <a href="#valuetype_number">number</a>, it defaults to 100.</li>
+				</ul>
+				<p>If authors do not set <code>aria-valuemin</code> and <code>aria-valuemax</code>, the value of <pref>aria-valuenow</pref> MUST NOT fall below or exceed the implicit <code>aria-valuemin</code> and <code>aria-valuemax</code> values, respectively.
```

Comment:
This seems to only apply when both attributes are omitted. But presumably the requirement should be `computed valuemin <= valuenow <= computed valuemax` where computed is the attribute's value if specified and not an error, otherwise the implicit value.